chemprop/cli/hpopt.py

```diff
@@ -253,10 +244,10 @@ def update_args_with_config(args: Namespace, config: dict) -> Namespace:
     for key, value in config.items():
         match key:
             case "final_lr_ratio":
-                setattr(args, "final_lr", value * config.get("max_lr", args.max_lr))
+                setattr(args, "final_lr", value * args.max_lr)
 
             case "init_lr_ratio":
-                setattr(args, "init_lr", value * config.get("max_lr", args.max_lr))
+                setattr(args, "init_lr", value * args.max_lr)
```

Comment:
Looks like there is a merge error here that set these lines back to a previous version.

Reply:
Thanks, you're right - this is from a merge error.

Reply:
I'll resolve the merge errors.